### PR TITLE
Adds pixel shifting

### PR DESCRIPTION
**Pull request by monster860**
originally an ERP pr on citadel. Certain people expressed interest in having it here. Not sure if it was sarcasm but I made a PR anyways. <https://github.com/Citadel-Station-13/Citadel-Station-13/pull/11349/files>

:cl:  
add: use CTRL+SHIFT + direction to shift your pixel
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
> originally an ERP pr on citadel. Certain people expressed interest in having it here. Not sure if it was sarcasm but I made a PR anyways. https://github.com/Citadel-Station-13/Citadel-Station-13/pull/11349/files
> 
> 
> use CTRL+SHIFT + direction to shift your pixel
> /

Now you can lean against bartops and tables..and walls...and other things!

**Comment by Potototatoh** [comment]
hover*

**Comment by deathslayer2481** [comment]
Urge to Purge intensifies.

**Comment by ThatLing** [comment]
PR doesn't work btw

**Review by ThatLing — CHANGES_REQUESTED** [review]
``code/modules/keybindings/bindings_mob.dm`` isn't used, read the first line of the file. Also, your changelog is broken

**Comment by LoliconSlayer** [comment]
Where's the fucking fetish content label

**Comment by overlord12345** [comment]
What dose this do im too dumb to understand

**Comment by Hopekz** [comment]
> What dose this do im too dumb to understand

This allows you to shift your character sprite 16 pixels in any direction so you can be closer to something that is not on your tile or you can co-exist on the same tile potentially.
Didn't test this yet but its a "why not" feature from me. 
Adds functionality and I can't see why not.

File: yogstation/code/modules/keybindings/bindings_mob.dm [hunk 1]
```diff
@@ -72,16 +72,28 @@
 
 		switch(dir)
 			if(NORTH)
-				northface()
+				if(client.keys_held["Shift"])
```
Comment (ThatLing) [inline yogstation/code/modules/keybindings/bindings_mob.dm]:
```
user.prefs.bindings.isheld_key("Shift")
```

**Review by ThatLing — APPROVED** [review]

**Pull request merged**